### DISCUSSION TITLE
Logo Squashed In Sidebar

### DISCUSSION THREAD

**Discussion by heather-timmins** [vikramlearning/blazorbootstrap#663] [Q&A]
Hi, 

My logo image is squashed in the side bar - How do i disable the title and allow my logo to be full width of the area? 

![image](https://github.com/vikramlearning/blazorbootstrap/assets/144691290/7c41fc4d-33f0-4710-8ba2-706c9480bdba)

**Comment by gvreddy04** [comment]
@heather-timmins Thank you for using BlazorBootstrap. Can you share a sample logo with some sample code that mimics your scenario? I'll take a look.
